packages/odyssey-react-mui/src/theme/ThemeContrastProvider.test.tsx

```diff
@@ -0,0 +1,158 @@
+/*!
+ * Copyright (c) 2024-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { render, screen, act } from "@testing-library/react";
+import { renderHook } from "@testing-library/react-hooks";
+import {
+  ThemeContrastProvider,
+  useThemeContrastContext,
+  getBackgroundColor,
+} from "../ThemeContrastProvider";
+import * as Tokens from "@okta/odyssey-design-tokens";
+import "@testing-library/jest-dom";
+
+jest.mock("../ThemeContrastProvider", () => ({
+  ...jest.requireActual("../ThemeContrastProvider"),
+  getBackgroundColor: jest.fn(),
+}));
```

Comment:
I don't think we need to mock `ThemeContrastProvider` in this test.

If we override what `getBackgroundColor` does, we can't be sure it's going to work that way when apps use it.

We should wrap this test in a `div` with a specific background color. That way, we can be sure it's working as-expected.